resources/dependencies.rb

```diff
@@ -17,6 +17,7 @@
 #
 
 action :install do
-  node.default['java']['jdk_version'] = '8'
-  include_recipe 'java'
+  openjdk_install '8' do
+    action :install
+  end
```

Comment:
It looks like openjdk isn't working but if you switch to adoptopenjdk it does work on Debian 9. I recommend you make that change.

In addition, this is a little cleaner way of using the resource.
```suggestion
  adoptopenjdk_install '8'
```